api/users/serializers.py

```diff
@@ -95,7 +95,7 @@ class Meta:
         read_only_fields = ("id",)
 
 
-class MyUserPermissionGroupsSerializer(serializers.ModelSerializer):
+class UserPermissionGroupNameSerializer(serializers.ModelSerializer):
```

Comment:
I think perhaps a better name might be `UserPermissionGroupSummarySerializer` ? But I'm ok with what you've got here too if you prefer.